apps/feedback/models/models.py

```diff
@@ -0,0 +1,84 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from wagtail.admin.panels import FieldPanel, MultiFieldPanel, HelpPanel, \
+    TitleFieldPanel, InlinePanel
+
+
+class Feedback(models.Model):
+    UNDECIDED = 0
+    APPROVED = 1
+    ANSWERED = 2
+    REJECTED = 3
+
+    STATUS_CHOICES = (
+        (UNDECIDED, _("Undecided")),
+        (APPROVED, _("Approved")),
+        (ANSWERED, _("Answered")),
+        (REJECTED, _("Rejected")),
+    )
+    answer = models.ForeignKey(
+        to="cms.Answer",
+        on_delete=models.SET_NULL,
+        null=True,
+    )
+
+    feedback = models.TextField(
+        verbose_name=_("Have a question or comment? Leave it here!"),
+        help_text=_("We will forward this response to the expert."),
+
+    )
+
+    grade = models.PositiveSmallIntegerField(
+        verbose_name=_(
+            "Do you find this article clearly explained and easy to follow?"
+        ),
+        help_text=_("(0 = not clear, 5 = very clear)"), null=True, blank=True
+    )
+    reliability_grade = models.PositiveSmallIntegerField(
+        verbose_name=_("Do you find this KlimaatHelpdesk article reliable?"),
+        help_text=_("(0 = not reliable, 5 = very reliable)"),
+    )
+
+    user_name = models.TextField(verbose_name=_("What is your name?"), null=True,
+                                 blank=True)
+    user_email = models.EmailField(
+        verbose_name=_("Email"),
+        help_text=_("We will forward the expert's response when we receive it."),
+        null=True,
```

Comment:
`null=True` hoeft nooit bij textvelden met blank. Blank zorgt er voor dat het db veld gevuld wordt met een lege string (`""`)

Reply:
Dit geldt voor alle text/char velden in dit model.